presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -334,6 +338,36 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+Table Statistics
+----------------
+
+The Hive connector automatically collects basic statistics
+(``numFiles', ``numRows``, ``rawDataSize``, ``totalSize``)
+on ``INSERT`` and ``CREATE TABLE AS`` operations.
+
+The Hive connector can also collect column level statistics:
+
+============= ================================================================================================================
+Column Type                                            Collectible Statistics
```

Comment:
Let's be consistent with other tables and put the "Collectible Statistics" at the start of the cell.
```
============= ====================================
Column Type   Collectible Statistics